server/pulp/server/webservices/http.py

```diff
@@ -71,7 +33,7 @@ def request_info(key):
     @rtype: str or None
     @return: request value
     """
-    return _get_wsgi_environ().get(key, None)
+    return _thread_local.wsgi_environ.get(key, None)
```

Comment:
This is resolved with the inclusion of SaveEnvironWSGIHandler in the last force push.